recipe/meta.yaml

```diff
@@ -59,7 +48,19 @@ outputs:
         - cccl =={{ cccl_version }}
       run_constrained:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    # Tests are defined at the top level, due to package/output name conflicts.
+    test:
+      requires:
+        - cmake  # [linux]
+      files:
+        - verify-version.cmake  # [linux]
+      commands:
+        - test -d $PREFIX/lib/cmake                      # [linux]
+        - test -d $PREFIX/targets/{{ target_name }}/lib  # [linux]
```

Comment:
Recognize this is primarily headers, but should we consider running `check-glibc` here (given we are copying over the libraries directory, which could include anything)